generating these info files on newer systems fails badly.  just ship a copy.

--- gcc/gcc.info
+++ gcc/gcc.info
@@ -0,0 +1,329 @@
+This is Info file gcc.info, produced by Makeinfo version 1.68 from the
+input file ./gcc.texi.
+
+INFO-DIR-SECTION Programming
+START-INFO-DIR-ENTRY
+* gcc: (gcc).                  The GNU Compiler Collection.
+END-INFO-DIR-ENTRY
+   This file documents the use and the internals of the GNU compiler.
+
+   Published by the Free Software Foundation 59 Temple Place - Suite 330
+Boston, MA 02111-1307 USA
+
+   Copyright (C) 1988, 1989, 1992, 1993, 1994, 1995, 1996, 1997, 1998,
+1999, 2000 Free Software Foundation, Inc.
+
+   Permission is granted to make and distribute verbatim copies of this
+manual provided the copyright notice and this permission notice are
+preserved on all copies.
+
+   Permission is granted to copy and distribute modified versions of
+this manual under the conditions for verbatim copying, provided also
+that the sections entitled "GNU General Public License" and "Funding
+for Free Software" are included exactly as in the original, and
+provided that the entire resulting derived work is distributed under
+the terms of a permission notice identical to this one.
+
+   Permission is granted to copy and distribute translations of this
+manual into another language, under the above conditions for modified
+versions, except that the sections entitled "GNU General Public
+License" and "Funding for Free Software", and this permission notice,
+may be included in translations approved by the Free Software Foundation
+instead of in the original English.
+
+
+Indirect:
+gcc.info-1: 1455
+gcc.info-2: 42284
+gcc.info-3: 76298
+gcc.info-4: 123574
+gcc.info-5: 167753
+gcc.info-6: 214257
+gcc.info-7: 261633
+gcc.info-8: 303250
+gcc.info-9: 350115
+gcc.info-10: 400095
+gcc.info-11: 441330
+gcc.info-12: 489450
+gcc.info-13: 530960
+gcc.info-14: 579852
+gcc.info-15: 627096
+gcc.info-16: 672202
+gcc.info-17: 722096
+gcc.info-18: 770877
+gcc.info-19: 817751
+gcc.info-20: 866577
+gcc.info-21: 916329
+gcc.info-22: 953390
+gcc.info-23: 995185
+gcc.info-24: 1040067
+gcc.info-25: 1085967
+gcc.info-26: 1130060
+gcc.info-27: 1179651
+gcc.info-28: 1219329
+gcc.info-29: 1264136
+gcc.info-30: 1288600
+
+Tag Table:
+(Indirect)
+Node: Top1455
+Node: G++ and GCC3216
+Node: Invoking GCC5614
+Node: Option Summary8975
+Node: Overall Options23720
+Node: Invoking G++28740
+Node: C Dialect Options30197
+Node: C++ Dialect Options42284
+Node: Warning Options59046
+Node: Debugging Options76298
+Node: Optimize Options93366
+Node: Preprocessor Options110838
+Node: Assembler Options117301
+Node: Link Options117668
+Node: Directory Options123574
+Node: Target Options127439
+Node: Submodel Options131070
+Node: M680x0 Options132620
+Node: VAX Options138136
+Node: SPARC Options138671
+Node: Convex Options148535
+Node: AMD29K Options150716
+Node: ARM Options154112
+Node: Thumb Options162133
+Node: MN10200 Options164103
+Node: MN10300 Options164627
+Node: M32R/D Options165403
+Node: M88K Options167753
+Node: RS/6000 and PowerPC Options175691
+Node: RT Options194805
+Node: MIPS Options196509
+Node: i386 Options206254
+Node: HPPA Options214257
+Node: Intel 960 Options218528
+Node: DEC Alpha Options221470
+Node: Clipper Options230864
+Node: H8/300 Options231263
+Node: SH Options232077
+Node: System V Options232896
+Node: TMS320C3x/C4x Options233714
+Node: V850 Options239219
+Node: ARC Options241228
+Node: NS32K Options242430
+Node: Code Gen Options246650
+Node: Environment Variables261633
+Node: Running Protoize267819
+Node: Installation274185
+Node: Configuration Files301645
+Node: Configurations303250
+Node: Other Dir342374
+Node: Cross-Compiler344090
+Node: Steps of Cross345921
+Node: Configure Cross347039
+Node: Tools and Libraries347676
+Node: Cross Runtime350115
+Node: Cross Headers354196
+Node: Build Cross356195
+Node: Sun Install358071
+Node: VMS Install359743
+Node: Collect2369673
+Node: Header Dirs372238
+Node: C Extensions373662
+Node: Statement Exprs377187
+Node: Local Labels379081
+Node: Labels as Values381143
+Node: Nested Functions383007
+Node: Constructing Calls386850
+Node: Naming Types388907
+Node: Typeof390001
+Node: Lvalues391866
+Node: Conditionals394306
+Node: Long Long395197
+Node: Complex396637
+Node: Hex Floats398498
+Node: Zero Length399418
+Node: Variable Length400095
+Node: Macro Varargs402620
+Node: Subscripting404723
+Node: Pointer Arith405206
+Node: Initializers405771
+Node: Constructors406236
+Node: Labeled Elements407930
+Node: Case Ranges410559
+Node: Cast to Union411240
+Node: Function Attributes412318
+Node: Function Prototypes428242
+Node: C++ Comments430044
+Node: Dollar Signs430580
+Node: Character Escapes431038
+Node: Alignment431325
+Node: Variable Attributes432797
+Node: Type Attributes441330
+Node: Inline450541
+Node: Extended Asm454783
+Node: Asm Labels469716
+Node: Explicit Reg Vars471035
+Node: Global Reg Vars472490
+Node: Local Reg Vars477055
+Node: Alternate Keywords478859
+Node: Incomplete Enums480261
+Node: Function Names481017
+Node: Return Address482291
+Node: Other Builtins484334
+Node: Deprecated Features486800
+Node: C++ Extensions488077
+Node: Naming Results489450
+Node: Min and Max492764
+Node: Destructors and Goto494204
+Node: C++ Interface494680
+Node: Template Instantiation499914
+Node: Bound member functions508529
+Node: C++ Signatures509862
+Node: Gcov514206
+Node: Gcov Intro514729
+Node: Invoking Gcov517410
+Node: Gcov and Optimization523021
+Node: Gcov Data Files524445
+Node: Trouble528015
+Node: Actual Bugs529693
+Node: Installation Problems530960
+Node: Cross-Compiler Problems544638
+Node: Interoperation546109
+Node: External Bugs559387
+Node: Incompatibilities561510
+Node: Fixed Headers569982
+Node: Standard Libraries572292
+Node: Disappointments573533
+Node: C++ Misunderstandings578072
+Node: Static Definitions578798
+Node: Temporaries579852
+Node: Copy Assignment581830
+Node: Protoize Caveats583641
+Node: Non-bugs587597
+Node: Warnings and Errors597228
+Node: Bugs598989
+Node: Bug Criteria600337
+Node: Bug Lists602775
+Node: Bug Reporting603882
+Node: Sending Patches616096
+Node: Service621471
+Node: Contributing622038
+Node: VMS622844
+Node: Include Files and VMS623230
+Node: Global Declarations627096
+Node: VMS Misc631401
+Node: Portability635708
+Node: Interface637457
+Node: Passes642039
+Node: RTL660390
+Node: RTL Objects662358
+Node: RTL Classes665404
+Node: Accessors669822
+Node: Flags672202
+Node: Machine Modes682537
+Node: Constants690171
+Node: Regs and Memory695359
+Node: Arithmetic707835
+Node: Comparisons713733
+Node: Bit Fields717796
+Node: Conversions719208
+Node: RTL Declarations722096
+Node: Side Effects722905
+Node: Incdec735909
+Node: Assembler739410
+Node: Insns740932
+Node: Calls764267
+Node: Sharing766862
+Node: Reading RTL769938
+Node: Machine Desc770877
+Node: Patterns772730
+Node: Example775674
+Node: RTL Template776802
+Node: Output Template789881
+Node: Output Statement793863
+Node: Constraints797576
+Node: Simple Constraints798579
+Node: Multi-Alternative810491
+Node: Class Preferences813327
+Node: Modifiers814207
+Node: Machine Constraints817751
+Node: No Constraints826813
+Node: Standard Names827934
+Node: Pattern Ordering865350
+Node: Dependent Patterns866577
+Node: Jump Patterns869392
+Node: Insn Canonicalizations875208
+Node: Peephole Definitions878703
+Node: Expander Definitions885621
+Node: Insn Splitting893006
+Node: Insn Attributes899937
+Node: Defining Attributes900984
+Node: Expressions902996
+Node: Tagging Insns909590
+Node: Attr Example913953
+Node: Insn Lengths916329
+Node: Constant Attributes919622
+Node: Delay Slots920782
+Node: Function Units923993
+Node: Target Macros929663
+Node: Driver931546
+Node: Run-time Target947093
+Node: Storage Layout953390
+Node: Type Layout970717
+Node: Registers977490
+Node: Register Basics978470
+Node: Allocation Order983266
+Node: Values in Registers984682
+Node: Leaf Functions989802
+Node: Stack Registers992580
+Node: Obsolete Register Macros993413
+Node: Register Classes995185
+Node: Stack and Calling1015192
+Node: Frame Layout1015647
+Node: Stack Checking1021144
+Node: Frame Registers1024724
+Node: Elimination1029139
+Node: Stack Arguments1033395
+Node: Register Arguments1040067
+Node: Scalar Return1050577
+Node: Aggregate Return1054815
+Node: Caller Saves1058530
+Node: Function Entry1060069
+Node: Profiling1070971
+Node: Varargs1078052
+Node: Trampolines1085967
+Node: Library Calls1092685
+Node: Addressing Modes1100743
+Node: Condition Code1111151
+Node: Costs1117350
+Node: Sections1130060
+Node: PIC1136291
+Node: Assembler Format1139000
+Node: File Framework1140066
+Node: Data Output1144764
+Node: Uninitialized Data1151958
+Node: Label Output1157364
+Node: Initialization1168570
+Node: Macros for Initialization1174713
+Node: Instruction Output1179651
+Node: Dispatch Tables1187838
+Node: Exception Region Output1190391
+Node: Alignment Output1193385
+Node: Debugging Info1195608
+Node: All Debuggers1196217
+Node: DBX Options1199045
+Node: DBX Hooks1204328
+Node: File Names and DBX1207968
+Node: SDB and DWARF1209941
+Node: Cross-compilation1212763
+Node: Misc1219329
+Node: Config1243706
+Node: Fragments1250969
+Node: Target Fragment1251565
+Node: Host Fragment1255720
+Node: Funding1256322
+Node: GNU/Linux1258821
+Node: Copying1264136
+Node: Contributors1283324
+Node: Index1288600
+
+End Tag Table
--- gcc/cpp.info
+++ gcc/cpp.info
@@ -0,0 +1,80 @@
+This is Info file cpp.info, produced by Makeinfo version 1.68 from the
+input file ./cpp.texi.
+
+INFO-DIR-SECTION Programming
+START-INFO-DIR-ENTRY
+* Cpp: (cpp).		       The GNU C preprocessor.
+END-INFO-DIR-ENTRY
+
+   This file documents the GNU C Preprocessor.
+
+   Copyright 1987, 1989, 1991, 1992, 1993, 1994, 1995, 1997, 1998 Free
+Software Foundation, Inc.
+
+   Permission is granted to make and distribute verbatim copies of this
+manual provided the copyright notice and this permission notice are
+preserved on all copies.
+
+   Permission is granted to copy and distribute modified versions of
+this manual under the conditions for verbatim copying, provided also
+that the entire resulting derived work is distributed under the terms
+of a permission notice identical to this one.
+
+   Permission is granted to copy and distribute translations of this
+manual into another language, under the above conditions for modified
+versions.
+
+
+Indirect:
+cpp.info-1: 928
+cpp.info-2: 50059
+cpp.info-3: 91244
+
+Tag Table:
+(Indirect)
+Node: Top928
+Node: Global Actions3837
+Node: Directives6357
+Node: Header Files8044
+Node: Header Uses8703
+Node: Include Syntax10195
+Node: Include Operation13337
+Node: Once-Only15199
+Node: Inheritance17624
+Node: Macros20157
+Node: Simple Macros21071
+Node: Argument Macros24059
+Node: Predefined29857
+Node: Standard Predefined30287
+Node: Nonstandard Predefined37945
+Node: Stringification41521
+Node: Concatenation44447
+Node: Undefining47720
+Node: Redefining48759
+Node: Macro Pitfalls50059
+Node: Misnesting51163
+Node: Macro Parentheses52177
+Node: Swallow Semicolon54045
+Node: Side Effects55943
+Node: Self-Reference57641
+Node: Argument Prescan59917
+Node: Cascaded Macros64919
+Node: Newlines in Args66064
+Node: Conditionals67409
+Node: Conditional Uses68761
+Node: Conditional Syntax70184
+Node: #if Directive70770
+Node: #else Directive73059
+Node: #elif Directive73726
+Node: Deleted Code75104
+Node: Conditionals-Macros76165
+Node: Assertions79850
+Node: #error Directive84085
+Node: Combining Sources85570
+Node: Other Directives88481
+Node: Output89935
+Node: Invocation91244
+Node: Concept Index106028
+Node: Index109040
+
+End Tag Table
